packages/core/src/Modules/Cashier/Components/Receipt/account-transfer-receipt.jsx

```diff
@@ -7,141 +7,141 @@ import { formatMoney, getCurrencyDisplayCode, getCurrencyName, routes } from '@d
 import { localize, Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 
-class AccountTransferReceipt extends React.Component {
-    state = {
-        is_switch_visible: false,
-        switch_to: {},
+const AccountTransferReceipt = ({
+    disableApp,
+    enableApp,
+    history,
+    receipt,
+    resetAccountTransfer,
+    selected_from,
+    selected_to,
+    switchAccount,
+}) => {
+    const [is_switch_visible, setIsSwitchVisible] = React.useState(false);
+    const [switch_to, setIsSwitchTo] = React.useState({});
+
+    React.useEffect(() => {
+        return () => {
+            resetAccountTransfer();
+        };
+    }, [resetAccountTransfer]);
+
+    const openStatement = () => {
+        history.push(routes.statement);
+        resetAccountTransfer();
     };
```

Comment:
Could we define this function outside of the component and pass required params?